src/containers/appointmentsPage/AppointmentsPage.js

```diff
@@ -1,3 +1,40 @@
+<<<<<<< HEAD

```

Comment:
This code is problematic - looks like it is related to a merge conflict, and both lots of code have been saved.

Reply:
We need bring in the array to TileList. We have done this section and merged to master Saba. You may want to review and take the code in master which has been tested working all fine.